pkg/manager/member/tikv_member_manager.go

```diff
@@ -284,6 +295,48 @@ func (tkmm *tikvMemberManager) syncTiKVConfigMap(tc *v1alpha1.TidbCluster, set *
 	return tkmm.typedControl.CreateOrUpdateConfigMap(tc, newCm)
 }
 
+func (tkmm *tikvMemberManager) syncTiKVAutoScalingConfigMap(tc *v1alpha1.TidbCluster) error {
```

Comment:
If auto-scaling is enabled, we would create and sync a new configmap only for the tikv created by auto-scaler.